.github/workflows/release.yml

```diff
@@ -163,3 +204,25 @@ jobs:
           path: dist
           merge-multiple: true
       - uses: pypa/gh-action-pypi-publish@release/v1
+
+  windows-installer-publish:
+    needs: [build_windows_installer, test_windows_installer]
+    name: Upload Windows installer
+    runs-on: ubuntu-latest
+    environment:
+      name: assets
```

Comment:
This links this job to the "assets" [deployment environment](https://github.com/silx-kit/silx/settings/environments) which requires validation before execution (in a similar way as for publishing on pypi)